src/user-federation/UserFederationLdapSettings.tsx

```diff
@@ -67,22 +131,13 @@ const LdapSettingsHeader = ({
           titleKey="LDAP"
           subKey=""
           dropdownItems={[
-            <DropdownItem
-              key="sync"
-              onClick={() => console.log("Sync users TBD")}
-            >
+            <DropdownItem key="sync" onClick={() => syncChangedUsers()}>
```

Comment:
no need to wrap it could just do:
```suggestion
            <DropdownItem key="sync" onClick={syncChangedUsers}>
```

Reply:
Thanks, yes agreed. Fixed these in the latest commit.